website/blog/2024-08-14-otel-context-propagation-in-ferretdb.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: otel-context-propagation-in-ferretdb
+title: 'OpenTelemetry context propagation in FerretDB'
+authors: [elena]
+description: >
+  In this blog post, we demonstrate how to pass tracing context to queries in FerretDB using OpenTelemetry.
+tags: [observability]
+---
+
+In today's world of distributed systems, achieving reliability depends on various factors, one of which is effective observability.
```

Comment:
```suggestion
In distributed systems, achieving reliability depends on various factors, one of which is effective observability.
```

Reply:
Let's fix content first, then review style and such